inc/template-tags.php

```diff
@@ -356,20 +358,27 @@ function responsive_primary_nav() {
  * If the current site has a site-wide ACL applied or the utility menu has
  * no items nothing will be displayed.
  *
+ * @param string $extra_classes Optional. Extra classes for the menu markup.
  * @param array $args {
  *     Optional. Arguments to configure menu markup.
  *
  *     @type  string $before HTML markup to display before menu.
  *     @type  string $after  HTML markup to display after menu.
  * }
  */
-function responsive_utility_nav( $args = array() ) {
+function responsive_utility_nav( $extra_classes = NULL, $args = array() ) {
```

Comment:
We could probably consolidate this parameter into the existing `$args`, and separate out like wp_nav_menu does:

```
$defaults = array(
    'container' => 'nav',
    'container_class' => 'utility-nav',
    'container_extra_class' => '',
    'menu_id' => 'utility-nav-menu',
    'menu_class' => 'utility-nav-menu',
);

$args = wp_parse_args( $args, $defaults );
```